test/dish_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+
+class DishTest < Minitest::Test
+
+  def test_basic_dish
```

Comment:
Seems like there's some duplication here. You are testing for the name and category here and the other tests below.